### ISSUE TITLE
Display time remaining for segmentation/extracting length

### ISSUE THREAD

**Issue by nesod1** [Abe404/root_painter#42]
When segmenting large datasets it would be useful if Rootpainter displayed 'Time remaining' or 'Predicted finish time' displayed as well as the % segmented, especially useful when it takes 24/48 hours to know when to check back!!

**Comment by Abe404** [comment]
Good idea! I will have a look into implementing this by measuring the time it took to segment the last 100 images (or however many have been segmented so far, if less than 100). Then I can calculate the estimated remaining time using the total images left to segment (which we already track).

**Comment by Abe404** [comment]
This has been implemented in the code but a release has not yet been made available. I will update you when a new client application is available to download. See screenshot for a preview:
<img width="504" alt="Screenshot 2022-09-16 at 14 17 21" src="https://user-images.githubusercontent.com/376295/190638017-ab5ef831-9d23-47e0-bc4d-f1707063c92f.png">


I implemented by just getting the time spent processing since the first image was processed and how many images are left to process.
